### PR TITLE
Fixes #36936 - Switch to terser

### PR DESCRIPTION
**Pull request by ehelms**
This includes temporary updates to packit in order to test this update against terser. This points to the scratch repo generated by the packaging PR (https://github.com/theforeman/foreman-packaging/pull/9877) and also points to a branch on my foreman-packaging fork that updates the spec file dependency.

### CODE REVIEW

File: bundler.d/assets.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
   gem 'gettext_i18n_rails_js', '~> 1.4'
   gem 'po_to_json', '~> 1.1'
   gem 'execjs', '>= 1.4.0', '< 3.0'
-  gem 'uglifier', '>= 1.0.3'
+  gem 'terser', '>= 1.1.0'
```
Comment (evgeni) [inline bundler.d/assets.rb]: any reason to use `>= 1.1.0` and not like Maria `~> 1.1`?

**Review by evgeni — DISMISSED** [review]
Should this get a Redmine?

**Comment by ehelms** [comment]
> Should this get a Redmine?

It already has one. Github just doesn't update things to make that obvious.

**Review by evgeni — APPROVED** [review]

**Comment by ehelms** [comment]
[test katello]

**Pull request merged**